index.html

```diff
@@ -1096,8 +1096,14 @@ <h2><dfn data-lt="implicitly release the pointer capture|implicitly releasing po
                 if the pointer supports hover, user agent MUST also send corresponding boundary events necessary
                 to reflect the current position of the pointer with no capture.</p>
             <p>When the <a>pointer capture target override</a> is no longer [=connected=] [[DOM]],
-                the <a>pending pointer capture target override</a> and <a>pointer capture target override</a> nodes SHOULD be cleared
-                and also a PointerEvent named {{GlobalEventHandlers/lostpointercapture}} corresponding to the captured pointer SHOULD be fired at the document.</p>
+                the <a>pending pointer capture target override</a> node SHOULD be cleared and
+                the <a>pointer capture target override</a> SHOULD be set to the document.</p>
+                <div class="note">
+                    This will result in a {{GlobalEventHandlers/lostpointercapture}} event corresponding to the captured pointer being fired at the document
```

Comment:
Another corner case: should we explicitly disallow `document.setPointerCapture`?

Reply:
Alternatively, we can say here:

> This will result in a `lostpointercapture` event corresponding to the captured pointer being fired at the document during the next Process pending pointer capture, _provided that there is no pending setPointerCapture request for the document itself_.

Reply:
Please ignore this thread, sorry: `document` is a non-element node, so it has no `setPointercapture`.